### ISSUE TITLE
Updates to VAMC Police data page after collab cycle feedback

### ISSUE THREAD

**Issue by thejordanwood** [department-of-veterans-affairs/va.gov-cms#15631]
## Description

- This ticket will be worked on after the VAMC Police page goes to Design Intent on 10/24/2023
- Updates will include:
   - Feedback from Design Intent
   - New police data details that have come in since completion of #15376 
   - Recommendation(s) for what to do in a data field when there isn't data available? Considerations:
     - [see Michelle's comments below](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15631#issuecomment-1819683399)
     - Leaving it blank may suggest police are hiding something
     - Putting a zero may suggest there is zero activity as opposed to there not being data available.

[Data elements](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/facilities/medical-centers/police/readme.md#data-elements)

## Design
[Sketch file with design](https://sketch.com/s/b48c905c-7c7f-46fb-9ae5-6020ff0e5174)

## Collab cycle tickets

- [Collab Cycle ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/65082)
- [IA feedback ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/68315)
- [Collab Cycle IA design](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/ia-design-docs/va-police-facilities-page.md)
- [CAIA centralized content](https://github.com/department-of-veterans-affairs/va.gov-team/issues/68949#issuecomment-1821152958) 
[67055 CAIA recommendations (1).docx](https://github.com/department-of-veterans-affairs/va.gov-cms/files/13428759/67055.CAIA.recommendations.1.docx)

## System Page v Facilities Pages 

The content model is being worked on for this. Currently, it looks like there will be a systems page with general information and the facilities pages would show the data. 

- [Content model ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15756)

## New Police Data Details

This feedback comes from comments left on #15376. 

- Michelle's feedback about current design ([Comment link](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15376#issuecomment-1747775733)) 
- Contact information updates ([Comment link](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15376#issuecomment-1753334054))

    > Physical Location
    > Telephone Numbers (24 hour dispatch number, external)
    > Chief of Police name and email
    > It is believed that this contact information is at the facility level but SME will confirm.

## Acceptance Criteria (Sprint 96)
- [x] Update design with Collab Cycle feedback 
- [x] Update design with new police data details
- [x] Determine what info shows on the systems page v facility pages

## Acceptance Criteria (Sprint 97) - 5 points
- [x] Update design with Collab Cycle feedback 
- [x] Review design with accessibility

## Acceptance Criteria (Sprint 98) - 8 points

- [x] Update design with CAIA feedback
- [x] @davidmpickett implement CAIA feedback in Drupal sources of truth / explain to CAIA why we aren't
- [x] Scheduled UX sync review with engineers and UX lead among others
- [ ] ~Notify product manager when design is ready for async collab cycle midpoint review~

## Acceptance Criteria (Sprint 99) - 5 points
- [x] Determine structure for arrest and ticketing data points 
- [x] Implement CAIA feedback after re-review
- [x] Update Sketch file and create updated mockups to reflect those changes
- [x] Review with UX lead
- [x] Review with accessibility

## Acceptance Criteria (Sprint 100) - 3 points
- [x] Once design is finalized @davidmpickett will update content doc and send both to CAIA with notes & Cue Jane to initiate async Midpoint Review
- [ ] ~~If Drupal is affected, note updates in #16329 (@davidmpickett )~~
- [x] Create a Mobile mockup

**Comment by jilladams** [comment]
Noting specifics from  #15376 re: Contact details: 

- Physical Location
- Telephone Numbers (24 hour dispatch number, external)
- Chief of Police name and email

**Comment by davidmpickett** [comment]
Updates from UX refinement - @thejordanwood won't be able to estimate points until after Design Intent. 

@jilladams @xiongjaneg ^ Candidate for sprint 96 with caveat of refinement happening during that sprint

**Comment by davidmpickett** [comment]
Did some digging and found the L4 pages aren't supposed to show the side nav, so we might be able to go that route for Police pages. For example, events pages have no side nav: 
- https://www.va.gov/amarillo-health-care/events/61955/
- technical details from way back when https://github.com/department-of-veterans-affairs/va.gov-team/issues/3575

**Comment by jilladams** [comment]
Mid-sprint: Getting reviews will take time, likely to roll over sprint boundary. Also holding for any CAIA feedback today.

**Comment by thejordanwood** [comment]
**Pre Out of Office update:**

I've created mockups in [the Sketch file](https://sketch.com/s/b48c905c-7c7f-46fb-9ae5-6020ff0e5174) to show the latest design decisions stated in [this Slack thread](https://dsva.slack.com/archives/C0FQSS30V/p1698966435539929). 

Changes include:
- Only need system-level page
- The list of facilities on the system page should turn into a drop-down
- Should have side nav and breadcrumbs

@aklausmeier will be filling in for any other design needs for this while I'm out.

**Comment by davidmpickett** [comment]
![VA Police with content input markup](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/51967950/dca2e5f1-c2c9-41c9-a4e3-90021e941f14)

**Comment by aklausmeier** [comment]
revised with additional information from Police Services and labeled with "Current V2 - Newest"

**Comment by thejordanwood** [comment]
@aklausmeier @davidmpickett I've made another revision for this and labeled it "Current V3 - Newest" in [the Sketch file](https://sketch.com/s/b48c905c-7c7f-46fb-9ae5-6020ff0e5174). Revisions were made based on comments that were on the V2 version in the file.

**Comment by davidmpickett** [comment]
Feedback from @laflannery in UX refinement:
Year and month filters should be grouped

**Comment by mmiddaugh** [comment]
Possible unhappy path scenarios...

- The specified facility has no police contact information or data 
- The specified facility has police contact information but police data is incomplete or unavailable, such as 
   - data is available for all but one or two data points  
   - data is entirely empty for the entire set of prosecution, professional standards, or use of force data 
   - there is no data at all
- The specified facility has police data but no police contact information 
- future: data is available for some time periods but not the selected period 

Question: should a facility without any data or contact info be available within the dropdown for that system?

Initial thoughts 
- if there is no data for an entire section, do not show that header or section at all 
- if a data point is missing, we should indicate no data is available (rather than saying "0") 
   - if space allows, suggest spelling out "not available" instead of n/a, to avoid being misinterpreted as "not applicable"

**Comment by aklausmeier** [comment]
@thejordanwood @mmiddaugh in addition, I was thinking we want to be prepared for the possibility of data temporarily being unavailable for technical reasons and being unable to show at this time but @mmiddaugh 's second bullet covers this scenario.

**Comment by jilladams** [comment]
Thanks! @thejordanwood @xiongjaneg I updated the ACs to reflect points per sprint for 97 and 98, and what I think I understood about what got done alst sprint based on comments, vs. what's left to do this sprint. Not 100% sure if Laura reviewed last sprint or needs to review this sprint, so please feel free to update as needed to make it make sense.

**Comment by xiongjaneg** [comment]
Updated ACs to reflect scheduled UX sync review and replace PO review with UX Lead review per new process

**Comment by davidmpickett** [comment]
I have updated the [Centralized Content node](https://prod.cms.va.gov/centralized-content/vamc-system-va-police-page-content) with many of the copy edits from CAIA. Here's a [side-by-side comparison](https://prod.cms.va.gov/node/63204/revisions/view/822129/824142/visual_inline)

**Comment by davidmpickett** [comment]
@xiongjaneg @mmiddaugh @thejordanwood @aklausmeier 

Here is a word doc where I have accepted all the proposed changes that implemented in Centralized Content (see previous comment). I also added a couple comments about changes that need to be implemented elsewhere. The bulk of the changes in the middle of the doc are more for Jordan to weave into Sketch as warranted

[67055.CAIA.recommendations.dmp.docx](https://github.com/department-of-veterans-affairs/va.gov-cms/files/13482137/67055.CAIA.recommendations.dmp.docx)

For ease of responding, here are my comments from inside the doc:
- [H1] VA P~~p~~olice
  - I noted this capitalization change in [Steve's PR](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/16169#discussion_r1406939636) 
- H1] VA police **at [facility name]**
  - We should not implement this. This goes against [the IA guidance](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/ia-design-docs/va-police-facilities-page.md).  
- [Additional info] What do these terms mean? (_second instance_)
  - This second block of definitions will need to be added to Centralized Content. I will stub a ticket for that

**Comment by mmiddaugh** [comment]
- I will request definitions for "use of force" and "weapons discharge"
- The [H4 ]"Arrests" in the document should be [H4] "Prosecutions", with "Ticketing" and "Arrests" subgroups
  - "Ticketing" has "Traffic and parking" and "Non-traffic (Criminal)" subgroups
- The [H4] "Reports of police misconduct" in the document should be [H4] "Professional Standards", with "Complaints and investigations", "Number of sustained allegations", and "Number of Disciplinary actions" as subgroups

**Comment by davidmpickett** [comment]
FYI - I am now officially pencils down on my AC for this Sprint

**Comment by davidmpickett** [comment]
Adding myself back so I can follow up on action item from UX sync to send CAIA an updated document to review

**Comment by davidmpickett** [comment]
@mmiddaugh @aklausmeier @xiongjaneg @thejordanwood 

I took a pass at an updated word doc based on the updated design. I made a couple comments in Sketch where the word doc is ahead. Can y'all take a look and see if this matches up before I send over to CAIA for re-review? 

[67055.CAIA.recommendations.dmp.docx](https://github.com/department-of-veterans-affairs/va.gov-cms/files/13550351/67055.CAIA.recommendations.dmp.docx)

**Comment by xiongjaneg** [comment]
@davidmpickett Looks good to me. I think you captured what I recalled from our conversations.

**Comment by aklausmeier** [comment]
@davidmpickett looks great including your comments in Word regarding changes on our side or reasoning on decisions

**Comment by davidmpickett** [comment]
Sent: https://github.com/department-of-veterans-affairs/va.gov-team/issues/68949

**Comment by mmiddaugh** [comment]
@davidmpickett - thank you for incorporating the latest updates - this should help with the re-review

One thought: 

**[H3] [month and year] police activity at [facility]** -> For this data, the context is "[number] calls for service `at this facility`" 

**[H3] [month and year] police activity at VA facilities nationwide** -> The context for this data is actually "[number] calls for service `for all facilities nationwide`". Given that we may not have access to this data, I suggest we remove that line for this data group, leaving only "This summary is current as of [month day, year]."

**Comment by thejordanwood** [comment]
@davidmpickett Thanks for catching some of the CAIA feedback that I missed! I've updated the Sketch file to reflect the changes that were needed.

**Comment by thejordanwood** [comment]
**My end of sprint update:**

I think I'm caught up on CAIA feedback for now. I'm working with @aklausmeier to determine how best to structure the arrest and ticketing data points. Once we figure that out, I'll update the Sketch file and create updated mockups to reflect those changes.

**Comment by jilladams** [comment]
Also: we have a last AC here to be ready for Midpoint, which we've opted to skip ([Slack thread](https://dsva.slack.com/archives/C0FQSS30V/p1701727780288809?thread_ts=1701725087.243679&cid=C0FQSS30V)). Striking through that AC.

**Comment by mmiddaugh** [comment]
Data definitions provided by Special Agent Steidler, unless otherwise noted 

Total number of calls – The number of times VA police have responded to incidents or requests for service (Michelle's definition)

- Prosecutions – Data about actions taken by VA police following traffic violations or criminal activity (Michelle's definition)
   - Arrests:  The number of times that VA Police take custody of a person to face criminal charges.  The arrest can be conducted on an existing warrant, or for crimes committed in the officers presence. 
   - Tickets: The number of times the VA police have given written notices following the violation of a law. Tickets can be issued for vehicle or parking violations, or crimes against a person or property.  Tickets can be issued for violations and can result in fines, or up to 6 months confinement.  

- Professional standards – Data about VA police conduct (Michelle's definition)
  - Complaints and investigations: The number of times VA has observed or received reports about the actions of a VA Police Officer that may violate VA Policies or regulations. 
  - Sustained allegations: The number of investigations about a VA Police Officer that found the officer violated VA Policies or regulations.
  - Disciplinary actions: The number of times VA has taken action to hold the VA Police Officer accountable for violating VA Policies or regulations that were sustained by investigations.

The following definitions are pending SME input
- Use of force
- Weapons discharge

FAQ(s) with more information about the Police Code of conduct and disciplinary actions are also pending

**Comment by thejordanwood** [comment]
@mmiddaugh @aklausmeier @davidmpickett I've cleaned up [the Sketch file ](https://sketch.com/s/b48c905c-7c7f-46fb-9ae5-6020ff0e5174)to better show the most recent version of the page. 

**Changes I made:**

- I've implemented the latest CAIA feedback, where relevant. Their feedback was for a past version and some of it no longer applied. 
- I added the definitions that Michelle suggested.
- I removed the first sentence under the "[month and year] police activity at VA facilities nationwide" header. I removed the line for the data group, leaving only "This summary is current as of [month day, year]."

**Changes left to make:**

- Use of force and Weapons discharge definitions will be added. (Definitions still pending)
- More FAQs will be added. (Info still pending)
- In the latest CAIA feedback, it was suggested that we move the contact section down. The [Word document](https://dvagov.sharepoint.com/:w:/s/SitewideCAIA/ESKYtD-cnh9Pg8rBf_kgYaoBmr8Mg7eMTrtKtKoGrXCDiA?e=IQyeOD) shows it right above the "[month and year] police activity at VA facilities nationwide" header. I wasn't sure if we wanted to move it down that far and have left it as is in the Sketch file. Is that a change I should make @aklausmeier?

**Comment by aklausmeier** [comment]
@thejordanwood I have concerns if someone is needing the phone number and we move it down, it becomes harder for the user to find. If they really need the number, we shouldn't bury it. Doesn't appear CAIA shared why it should be moved down. 

@davidmpickett Would love to know your thoughts on pushing back on this unless we have good reasoning.

**Comment by davidmpickett** [comment]
@aklausmeier @thejordanwood @mmiddaugh @xiongjaneg 

**Re: position of the "How to Contact us" section.** 
- Based on my work cleaning up the [phone number csv](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16335#issuecomment-1843785576), I think this should actually be **promoted from H3 to H2** and move above "How to request a VA police report"
- **Rationale**: the phone number information is primarily system-level information. Out of 140 VAMC systems, only 10 of them have more than one phone number per system.
- For mmVP, we will be having our Police stakeholders choose one phone number per system for the 10 systems that have multiple. 
- Even for MVP, I think the contact information will be better up here. 92% of systems will still just have one phone number. For the other 8%, it would probably make sense to always show multiple phone numbers up top rather than having phone numbers update per facility in the "Get police activity information"
- Semantically, contact information doesn't really fall under "Get police activity information" It's an entirely different category of content.
- From a Veteran perspective, we think the contact information is one of the most valuable parts of this page. That is why it's being prioritized for mmVP

**Comment by aklausmeier** [comment]
@davidmpickett Thank you for the added context. It would be good to share this reasoning with CAIA. At minimum it should be pulled out and above "Get police ..." so it stays at the system level even when a facility is selected.

**Comment by davidmpickett** [comment]
@xiongjaneg @jilladams FYI Added myself back on the ticket and upped the points and added an AC specific to my work

**Comment by davidmpickett** [comment]
@jilladams @xiongjaneg  Noting that my AC is blocked until design is finalized and unlikely to complete this sprint given that there are open questions. It was a mid-sprint addition, so wasn't officially part of sprint plan.

**Comment by thejordanwood** [comment]
I've made updates to [the Sketch file](https://sketch.com/s/b48c905c-7c7f-46fb-9ae5-6020ff0e5174).

**Changes I made:**

- Added Use of force and Weapons discharge definitions
- Split definitions up into 3 additional info components that correspond with each data section. 
- Moved How to contact us section above the How to request a police report section 

**Changes left to make:**

- More FAQs will be added. (Info still pending)

**Comment by jilladams** [comment]
@thejordanwood @davidmpickett (cc @xiongjaneg ) I took a pass at updating ACs here for what needs to happen in next sprint. Amanda noted in 1:1 that she's satisfied with our changes so far so doesn't need to re-review before we finalize / send back to CAIA. I"m not certain who owns the action around more FAQs to be added / if that's the same as Dave's AC for sending content to CAIA. Y'all please feel free to edit ACs as needed.

Note: team goal to get this back to CAIA when we can -- if possible to get back to them before holiday, that would be nice. Though we know most CAIA folks won't be working during the break, so we may not get much back until Jan.

**Comment by thejordanwood** [comment]
I've created a mobile mockup for the default/system page in [the Sketch file](https://sketch.com/s/b48c905c-7c7f-46fb-9ae5-6020ff0e5174). 

cc: @xiongjaneg

**Comment by davidmpickett** [comment]
I am currently working on updating the [content doc on Sharepoint] to reflect the current design (https://dvagov.sharepoint.com/:w:/s/SitewideCAIA/Ec-X8t6D9ZJPgoB085oaA2gBSA4iE4S0ixgjbIlJvaiCYw?e=1wUx7m)

**Comment by davidmpickett** [comment]
@mmiddaugh @aklausmeier, I have updated the [content doc on Sharepoint](https://dvagov.sharepoint.com/:w:/s/SitewideCAIA/Ec-X8t6D9ZJPgoB085oaA2gBSA4iE4S0ixgjbIlJvaiCYw?e=1wUx7m) to match the latest design. 
- I left a detailed comment about rationale for promoting contact info to H2 and bringing to top of page
- I took a stab at revising the term descriptions to match the CAIA style (embedded in a sentence as opposed to . Would appreciate @mmiddaugh's review to make sure these are accurate before we ping CAIA
-  @aklausmeier I left a comment in Figma on the subhead for the third data box. Reusing "police conduct" would be a semantic problem. The word doc reflects my recommendation.
- Tagged @mmiddaugh on some outstanding comment threads about stakeholder clarifications

**Comment by aklausmeier** [comment]
@davidmpickett 

- Agree with subhead, updated Sketch file to reflect change. My only concern would be if this pushes subhead to 3 lines once page is live, possibly dropping "incidents" if needed. 
- I'm left wondering if we still need "Prosecutions" header above "Arrests" and "Tickets". Is this causing unnecessary confusion (from Katherine's comment in Word doc)? Arrests and tickets are forms of prosecution, prosecution is not an action after an arrest or ticket. 
- I left a comment in Sketch for "Professional standards" definition, suggest replacing with "Police conduct" definition. 

cc @thejordanwood @mmiddaugh

**Comment by thejordanwood** [comment]
Design updates:

- I've updated the Sketch file to show correct "On this page" links. 
- @aklausmeier and I decided to remove the card outline around the data sections. It caused tight spacing on mobile between the data boxes/card outline and we felt it was unnecessary now that some headings have been removed or better defined. That's been updated in the Sketch file as well. 

cc: @davidmpickett @mmiddaugh

**Comment by davidmpickett** [comment]
Now I wish we had looked at this together in UX Sync. 😅 

Taking away the boxes also has implications for the content ordering. Without the box, it's not clear that Arrests and Tickets are subcategories of Prosecutions. If we're not going to have the boxes, then I think we should just have one level of headings. Would suggest removing Prosecutions all together and just having Arrests and Tickets as H4s.

Also then the pattern of the "what do these terms mean" should be one additional info component per set of cards (4 total, rather than 3). When Arrests and Tickets were combined in box, it make sense to have a unified additional info component. Now semantically, the additional info component seems to only be associated with the Tickets H5.

**Comment by davidmpickett** [comment]
I responded to CAIA to give them an update on project status: https://github.com/department-of-veterans-affairs/va.gov-team/issues/68949#issuecomment-1866944910

**Comment by davidmpickett** [comment]
@jilladams @xiongjaneg @mmiddaugh @aklausmeier 
Now that we are planning to take this design to Midpoint Review, this AC doesn't really make sense for this sprint:
- [ ] If Drupal is affected, note updates in https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16329 (@davidmpickett )
Really this should wait until after Midpoint review.

**Comment by jilladams** [comment]
@davidmpickett @thejordanwood Am I right that re: the remaining Sprint 100 AC (Once design is finalized @davidmpickett will update content doc and send both to CAIA with notes & Cue Jane to initiate async Midpoint Review)  that the questions in [this comment](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15631#issuecomment-1866931554)  need design / UX follow up first?

**Comment by thejordanwood** [comment]
@jilladams There's been [a Slack thread](https://dsva.slack.com/archives/C0FQSS30V/p1703114920925469) discussing the feedback in that comment. Here are the updates:

- We reached a consensus in the Slack thread and decided to remove the Prosecution header. I've removed that header and adjusted the Arrests and Tickets headers to h4s.
- Not discussed the Slack thread: I don't believe we need another additional info component, but think we should rename the ones we have. As Laura has pointed out previously, for accessibility reasons they don't all need to say "What do these terms mean?" We could specify and say something like. "What do these arrests and ticket terms mean?" I'm hoping that CAIA can provide some guidance on how to name these. 

cc: @davidmpickett @aklausmeier
